blackbox/docs/general/builtins/scalar.rst

```diff
@@ -2338,6 +2338,32 @@ Example::
     +--------------------+
     SELECT 1 row in set (... sec)
 
+.. _pg_typeof:
+
+``pg_typeof(expression)``
```

Comment:
any reason not to remove the `(expression)` part like I suggested?

Reply:
apologies, done